sky/data/mounting_utils.py

```diff
@@ -98,6 +163,26 @@ def get_cos_mount_cmd(rclone_config_data: str, rclone_config_path: str,
     return mount_cmd
 
 
+def _get_mount_binary(mount_cmd: str) -> str:
```

Comment:
what is this function for? Seems like it is just returning the mount cmd

Reply:
It's used to extract the mounting binary given the mounting command from `get_mounting_script`.

Reply:
Why not using the original one?

https://github.com/skypilot-org/skypilot/blob/feaaa3a21c16c0d8870f209020dad6de1a009454/sky/data/mounting_utils.py#L125